internal/cmd/beta/server/create/create.go

```diff
@@ -0,0 +1,339 @@
+	cmd.Flags().StringP(nameFlag, "n", "", "Server name")
+	cmd.Flags().String(machineTypeFlag, "", "Name of the type of the machine for the server. Possible values are documented in https://docs.stackit.cloud/stackit/en/virtual-machine-flavors-75137231.html")
+	cmd.Flags().String(affinityGroupFlag, "", "The affinity group the server is assigned to")
+	cmd.Flags().String(availabilityZoneFlag, "", "The availability zone of the server")
+	cmd.Flags().String(bootVolumeSourceIdFlag, "", "ID of the source object of boot volume. It can be either an image or volume ID")
+	cmd.Flags().String(bootVolumeSourceTypeFlag, "", "Type of the source object of boot volume. It can be either  'image' or 'volume'")
+	cmd.Flags().Int64(bootVolumeSizeFlag, 0, "The size of the boot volume in GB. Must be provided when 'boot-volume-source-type' is 'image'")
+	cmd.Flags().String(bootVolumePerformanceClassFlag, "", "Boot volume performance class")
+	cmd.Flags().Bool(bootVolumeDeleteOnTerminationFlag, false, "Delete the volume during the termination of the server. Defaults to false")
+	cmd.Flags().String(imageIdFlag, "", "The image ID to be used for an ephemeral disk on the server. Either 'image-id' or 'boot-volume-...' flags are required")
+	cmd.Flags().String(keypairNameFlag, "", "The name of the SSH keypair used during the server creation")
+	cmd.Flags().StringToString(labelFlag, nil, "Labels are key-value string pairs which can be attached to a server. E.g. '--labels key1=value1,key2=value2,...'")
+	cmd.Flags().String(networkIdFlag, "", "ID of the network for the initial networking setup for the server creation")
+	cmd.Flags().StringSlice(networkInterfaceIdsFlag, []string{}, "List of network interface IDs for the initial networking setup for the server creation")
+	cmd.Flags().StringSlice(securityGroupsFlag, []string{}, "The initial security groups for the server creation")
+	cmd.Flags().StringSlice(serviceAccountEmailsFlag, []string{}, "List of the service account mails")
+	cmd.Flags().Var(flags.ReadFromFileFlag(), userDataFlag, "User data that is passed via cloud-init to the server")
+	cmd.Flags().StringSlice(volumesFlag, []string{}, "The list of volumes attached to the server")
+
+	err := flags.MarkFlagsRequired(cmd, nameFlag, machineTypeFlag)
+	cmd.MarkFlagsMutuallyExclusive(imageIdFlag, bootVolumeSourceIdFlag)
+	cmd.MarkFlagsMutuallyExclusive(imageIdFlag, bootVolumeSourceTypeFlag)
+	cmd.MarkFlagsMutuallyExclusive(networkIdFlag, networkInterfaceIdsFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &cliErr.ProjectIdError{}
+	}
+
+	bootVolumeSourceId, _ := cmd.Flags().GetString(bootVolumeSourceIdFlag)
+	bootVolumeSourceType, _ := cmd.Flags().GetString(bootVolumeSourceTypeFlag)
+	bootVolumeSize, _ := cmd.Flags().GetInt64(bootVolumeSizeFlag)
+	imageId, _ := cmd.Flags().GetString(imageIdFlag)
```

Comment:
We should use the same utils that we also below for getting the values (e.g. `flags.FlagToStringPointer(p, cmd, bootVolumeSourceIdFlag)`)

Reply:
For uniformity but also as they already get the printer and add debug logs in case of errors, etc

Reply:
I added utils usage to aling with configureFlags. Also added some conditional checks more